src/main/frontend/app/components/ProductShowComponent.js

```diff
@@ -4,15 +4,19 @@ import '../css/showpage.scss'
 const ProductShowComponent = (props) => {
   return (
     <div className="flex-container">
-      <div><img src={props.image} /></div>
+      <div><img src={props.image} alt={props.name}/></div>
       <div>
         <div className="product-name">
           <h2>{props.name}</h2>
         </div>
         <h2>Price: ${props.price} & Free Shippping</h2>
         <p><a href={props.url}>Visit Product Page</a></p>
         <div className="description">
-          <p>Product Description: <br />{props.description}</p>
+          <p>Product Description:</p>
+          <br/>
```

Comment:
one thing to consider - you can add additional whitespace here with css making the `<br/>` tag unnecessary